mlflow/metrics/genai/prompts/v1.py

```diff
@@ -17,33 +17,50 @@
 }
 grading_system_prompt_template = PromptTemplate(
     """
-Please act as an impartial judge and evaluate the quality of the provided output which
-attempts to produce output for the provided input based on a provided information.
-
-You'll be given a grading format below which you'll call for each provided information,
-input and provided output to submit your justification and score to compute the {name} of
-the output.
-
-Input:
+====
+Task
+====
+You are an impartial judge. You will be given an input that was sent to a machine
+learning model, and you will be given an output that the model produced.
+There is additional information that was provided to the LLM to generate the output.
+
+Your task is to determine a numerical score called {name} based on the input and output.
+A definition of {name} and a grading rubric are provided below.
+You must use the grading rubric to determine your score. You must also justify your score.
+
+Examples are included below for reference. You must read them and make sure you
```

Comment:
"You must read them" is a bit funny lol, probably you can use "please make sure to use them as reference when completing the task"? This will force the judge to refer to the content when generating output

Reply:
Saying please to an LLM also seems funny